data/tech/aria/aria-colcount_attribute.json

```diff
@@ -9,6 +9,46 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#aria-colcount"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_column_count_all_cols",
```

Comment:
I'm wondering if we can remove this expectation. See my similar comment for `aria-rowcount` for context.

Reply:
Removed the all_cols expectation and test results. ✔️